.github/workflows/release.yml

```diff
@@ -0,0 +1,25 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK 11
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: 'sbt'
+      - name: Build tarball
+        run: sbt createTarball
+      - name: Upload Release Asset
+        uses: actions/upload-artifact@v2
+        with:
+         name: unitycatalog
+         path: target/unitycatalog-*.tar.gz
```

Comment:
this is for the release that runs only on the. can you add a regular test that ensure that tarball generation is never broken, and the generated tarball can be always untar-ed and use to start a UC server?

basically an overall tests for all the scripts